androidApp/src/androidMain/AndroidManifest.xml

```diff
@@ -13,6 +13,7 @@
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
+        android:name="org.jetbrains.kotlinconf.android.KotlinConfApplication"
```

Comment:
Please make sure the name and location is matching app location for the previous year, so we will be able to publish an update

Reply:
This here is only the reference to the application class used, it won't affect anything external-facing like the package name.